changelog/@unreleased/pr-62.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: fix
+fix:
+  description: |-
+    Exclude the last heartbeat timestamp from heartbeat params.
```

Comment:
Let's update this to reflect the new changes.

Reply:
done